packages/react-strict-dom/src/native/modules/createStrictDOMComponent.js

```diff
@@ -795,10 +796,8 @@ export function createStrictDOMComponent<T, P: StrictProps>(
       if (letterSpacing != null) {
         nextInheritedStyles.letterSpacing = letterSpacing;
       }
-      if (flatStyle.lineHeight != null) {
-        // Intentionally use the unresolved value to account for unitless
-        // lineHeight, which needs to be preserved when inherited.
-        nextInheritedStyles.lineHeight = flatStyle.lineHeight;
+      if (lineHeight != null) {
+        nextInheritedStyles.lineHeight = lineHeight;
```

Comment:
A [previous patch](ef62857d0b8c3ce1578b03dbe581e243f21c0977) changed this logic and caused a regression when computing em-unit lineHeights that are inherited by an element setting em-unit fontSize.